CHECKLIST.md

```diff
@@ -0,0 +1,55 @@
+# Checklist
+
+- [ ] I'm using the latest version of the [assembly project template](https://github.com/NBISweden/assembly-project-template)
+
+## Read data
+
+- [ ] All data delivered to us for the analysis are read-only in the folder `data/deliveries`.
+- [ ] The data used in the assembly are symlinked under `data/raw-data`.
+- [ ] Read metadata has also been received.
+
+## Analyses
+
+- [ ] All code necessary to create the final assembly is under `analyses` or `code`.
+- [ ] The file `analyses/History.md` has a step by step description of how to use the read data to make the submitted assembly.
```

Comment:
is this something we need to write or added automatically after each steps of the pipeline?

Reply:
ah yes I see it now